### DISCUSSION TITLE
Throw error when no workspace is found

### DISCUSSION THREAD

**Discussion by abdelp** [vercel/turborepo#7451] [Ideas]
### Which project is this feature idea for?

Turborepo

### Describe the feature you'd like to request

Hi everyone,
The specific scenario, for my case, is for GitHub actions, I have some workflows that run tests for different workspaces, and the problem is when there are some misspelled workspaces the execution silently ignores them and succeeds even though it didn't find any workspaces.

### Describe the solution you'd like

If you run:
```sh
yarn turbo run test --filter=graphl-api # wrong workspace name
```
instead of
```sh
yarn turbo run test --filter=graphql-api
```
it should abort the execution maybe with a flag `--strict` or something like that.

### Describe alternatives you've considered

The alternative that I considered is to save the output of the command and check its content to see if it matches:
`Running test in 0 packages`

**Comment by chris-olszewski** [comment]
Hi, we're currently in the process of deciding how we want to better signal possible typos in filters without adding another flag. Will communicate back on this issue once we come to a decision.

**Comment by anthonyshew** [comment]
Added to Turborepo 2.0 in this PR: https://github.com/vercel/turbo/pull/8142
